packages/plugins/nodejs-react-json-tree/nodejs-react-json-tree.spec

```diff
@@ -0,0 +1,110 @@
+Source15: https://registry.npmjs.org/setimmediate/-/setimmediate-1.0.5.tgz
+Source16: https://registry.npmjs.org/promise/-/promise-7.3.1.tgz
+Source17: https://registry.npmjs.org/isomorphic-fetch/-/isomorphic-fetch-2.2.1.tgz
+Source18: https://registry.npmjs.org/ua-parser-js/-/ua-parser-js-0.7.17.tgz
+Source19: https://registry.npmjs.org/asap/-/asap-2.0.6.tgz
+Source20: https://registry.npmjs.org/whatwg-fetch/-/whatwg-fetch-2.0.3.tgz
+Source21: https://registry.npmjs.org/node-fetch/-/node-fetch-1.7.3.tgz
+Source22: https://registry.npmjs.org/is-stream/-/is-stream-1.1.0.tgz
+Source23: https://registry.npmjs.org/encoding/-/encoding-0.1.12.tgz
+Source24: https://registry.npmjs.org/iconv-lite/-/iconv-lite-0.4.19.tgz
+Source25: react-json-tree-0.11.0-registry.npmjs.org.tgz
+BuildRequires: nodejs-packaging
+BuildArch:  noarch
+ExclusiveArch: %{nodejs_arches} noarch
+
+Provides: npm(%{npm_name}) = %{version}
+Provides: bundled(npm(react-json-tree)) = 0.11.0
+Provides: bundled(npm(react-base16-styling)) = 0.5.3
+Provides: bundled(npm(babel-runtime)) = 6.26.0
+Provides: bundled(npm(prop-types)) = 15.6.1
+Provides: bundled(npm(base16)) = 1.0.0
+Provides: bundled(npm(lodash.curry)) = 4.1.1
+Provides: bundled(npm(lodash.flow)) = 3.5.0
+Provides: bundled(npm(pure-color)) = 1.3.0
+Provides: bundled(npm(regenerator-runtime)) = 0.11.1
+Provides: bundled(npm(loose-envify)) = 1.3.1
+Provides: bundled(npm(object-assign)) = 4.1.1
+Provides: bundled(npm(fbjs)) = 0.8.16
+Provides: bundled(npm(js-tokens)) = 3.0.2
+Provides: bundled(npm(core-js)) = 2.5.3
+Provides: bundled(npm(core-js)) = 1.2.7
+Provides: bundled(npm(setimmediate)) = 1.0.5
+Provides: bundled(npm(promise)) = 7.3.1
+Provides: bundled(npm(isomorphic-fetch)) = 2.2.1
+Provides: bundled(npm(ua-parser-js)) = 0.7.17
+Provides: bundled(npm(asap)) = 2.0.6
+Provides: bundled(npm(whatwg-fetch)) = 2.0.3
+Provides: bundled(npm(node-fetch)) = 1.7.3
+Provides: bundled(npm(is-stream)) = 1.1.0
+Provides: bundled(npm(encoding)) = 0.1.12
+Provides: bundled(npm(iconv-lite)) = 0.4.19
+AutoReq: no
+AutoProv: no
+
+
+%define npm_cache_dir /tmp/npm_cache_%{name}-%{version}-%{release}
+%description
+%{summary}
+
+%prep
+mkdir -p %{npm_cache_dir}
+for tgz in %{sources}; do
+  echo $tgz | grep -q registry.npmjs.org || npm cache add --cache %{npm_cache_dir} $tgz
+done
+
+%setup -T -q -a 25 -D -n %{npm_cache_dir}
+
+%build
+npm install --cache-min Infinity --cache %{npm_cache_dir} --no-optional --global-style true %{npm_name}@%{version}
+
+%install
+mkdir -p %{buildroot}%{nodejs_sitelib}/%{npm_name}
+cd node_modules/react-json-tree
+cp -pfr CHANGELOG.md CODE_OF_CONDUCT.md LICENSE.md README.md lib package.json yarn.lock node_modules %{buildroot}%{nodejs_sitelib}/%{npm_name}
+cp -pf CHANGELOG.md CODE_OF_CONDUCT.md LICENSE.md README.md LICENSE.md ../../
+
+%clean
+rm -rf %{buildroot} %{npm_cache_dir}
+
+%if 0%{?enable_tests}
+%check
+%{nodejs_symlink_deps} --check
+%endif
+
+%files
+%{nodejs_sitelib}/%{npm_name}
+
+%license LICENSE.md
```

Comment:
In my updated template I started to use `%license node_modules/%{npm_name}/LICENSE.md`. That way you don't have to copy them in `%install`. This is also true for all the `%doc` files below. In the actual RPM/installed files there's no difference between the two.